### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by roigcarlo**
**Description**
Update Zenodo reference

**Changelog**
- Update Zenodo Reference

### CODE REVIEW

**Review by riccardotosi — DISMISSED** [review]

**Review by loumalouomega — DISMISSED** [review]
Okay

**Comment by loumalouomega** [comment]
Typo : I wrote batch and is badge

**Review by philbucher — DISMISSED** [review]

**Review by loumalouomega — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -78,6 +78,11 @@ Some users of the technologies developed in Kratos are:
 <img align="left" src="https://github.com/KratosMultiphysics/Documentation/raw/master/Wiki_files/Logos/onera_logo.png" width="128">
 <p>ONERA, The French Aerospace Lab<br>Applied Aerodynamics Department</p>
 </span>
+<span>
+<img align="left" src="https://github.com/KratosMultiphysics/Documentation/raw/master/Wiki_files/Logos/Deltares_logo.png" width="128">
```
Comment (loumalouomega) [inline README.md:82]: Can I say the resolution is not very HD?

Comment (roigcarlo) [reply README.md:82]: Its is not, Its optimized for 128 pixels

**Pull request merged**